corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -199,13 +199,17 @@ hqDefine("cloudcare/js/formplayer/router", function () {
         API.listMenus();
     });
 
-    FormplayerFrontend.on("menu:query", function (queryDict, selectValuesByKeys = false) {
+    FormplayerFrontend.on("menu:query", function (queryDict, selectValuesByKeys = false, forceManualSearch) {
         var urlObject = utils.currentUrlToObject();
-        urlObject.setQueryData({
-            inputs: queryDict,
-            execute: true,
-            selectValuesByKeys: selectValuesByKeys,
-        });
+        var queryObject = _.extend(
+            {
+                inputs: queryDict,
+                execute: true,
+                selectValuesByKeys,
+            },
+            forceManualSearch ? { forceManualSearch: true } : {}
```

Comment:
nit: was there a need for the new forceManualSearch var in `query.js`? Could have we just passed in sidebarEnabled and then in line 210 check if  that var is true? Again very nit

Reply:
Good point, I don't think it really is necessary.